packages/core/src/external/carequality/domain/document-query-result.ts

```diff
@@ -0,0 +1,9 @@
+import { DocumentQueryResponseOutgoing } from "@metriport/ihe-gateway-sdk";
+import { BaseResultDomain } from "./ihe-result";
+
+export const TABLE_NAME = "document_query_result";
+export const REQUEST_ID_COLUMN = "request_id";
```

Comment:
nit: this is model related, so maybe move to the model file?

Reply:
This is so it can be used on core

Reply:
Gotcha, b/c of the raw query we do on the lambda.

Yeah, ideally we wouldn't have table/DB related stuff on code that's used on the service/domain layer - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#9b81f21f7496456aa255eae3be3b617a)

So maybe we have these just as constants on the actual file called by the Lambda code?